commands.json

```diff
@@ -22,7 +22,8 @@
     "less",
     "grep",
     "rm",
-    "unalias"
+    "unalias",
+    "which"
     ],
   "globalExclusions": [
     "alias",
```

Comment:
@khuongduybui yeah, gulp should generate everything for you. It should be easy--if it's not, let us know and we can work on fixing it.